test/integration/istio_http_integration_test.cc

```diff
@@ -51,6 +52,22 @@ constexpr char kGoodToken[] =
     "XOxJ3Uc1MI0tz3uMiiZcyPV7sNCU4KRnemRIMHVOfuvHsU60_GhGbiSFzgPT"
     "Aa9WTltbnarTbxudb_YEOx12JiwYToeX0DCPb43W1tzIBxgm8NxUg";
 
+// Generate by gen-jwt.py as described in
+// https://github.com/istio/istio/blob/master/security/tools/jwt/samples/README.md
+// to generate token with another issuer.
+// `security/tools/jwt/samples/gen-jwt.py security/tools/jwt/samples/key.pem
+//  --expire=3153600000 --claims=foo:bar --iss "testing2@secure.istio.io"`
```

Comment:
[optional] You may change the claims to make it more diverge. Even change issuer to smt totally different, so it's easy to spot in the log output.

Reply:
Done, updated the issuer to `testing-rbac`